src/components/forms/Input.tsx

```diff
@@ -5,7 +5,7 @@ import { FormikErrorMessage } from "./index";
 import classNames from "classnames";
 
 const defaultInputClassNames =
-  "focus:ring-sdb-100 focus:border-sdb-100 block border-gray-300 rounded-md disabled:opacity-75 disabled:cursor-not-allowed";
```

Comment:
I think you added block when you added sanger-ui styling package. Maybe devel needs to be merged into this branch?

Reply:
Yes, I will resolve conflicts soon